core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/CachableBlockFile.java

```diff
@@ -69,11 +66,10 @@ public static String pathToCacheId(Path p) {
 
   public static class CachableBuilder {
     String cacheId = null;
-    IoeSupplier<InputStream> inputSupplier = null;
+    IoeSupplier<FSDataInputStream> inputSupplier = null;
```

Comment:
What was the reason for changing this type?

Reply:
The BCFile.Reader constructor takes `InputStreamType extends InputStream & Seekable` as an argument, not a plain `InputStream`. I don't think I can cast an `InputStream` to this type when I call the constructor, so I have to make it a type that implements both of these things.